### DISCUSSION TITLE
How to start

### DISCUSSION THREAD

**Discussion by Cheems-arch** [community/community#129108] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

I just started github. I don't really know anything. I saw some github commands online. Do help on starting.

**Comment by Rishi-Sudhakar** [comment, marked as answer]
To start with GitHub as a beginner, first, create an account and install Git on your computer. Begin by exploring some public repositories to get a feel for how projects are structured. Use the GitHub Desktop app for an easier start with version control, cli is good too (just saying). Practice by creating your own repository, making commits, and pushing changes. Don't hesitate to contribute to open-source projects when you gain more experience; it's a great way to learn. Check out GitHub's guides and tutorials for step-by-step instructions(YouTube is the best way to learn), and remember, the community is friendly and always ready to help you grow and learn. Happy coding!

**Reply by Cheems-arch** [reply]
Thank you so much

**Reply by Transparency-66** [reply]
My device is hacked, and who is using and management my personal information and devices control my life since 2019 and more years.

**Reply by Transparency-66** [reply]
I need legal help out of Arizona because it's hard to send information.

**Reply by Mo757484** [reply]
♥️♥️

**Reply by Rakib-Nasim** [reply]
if i want to share my github profile link with anybody , and he or she click on my profile link he  or she get github 404 this is not the webpage you are looking for .... what is the problem?

**Comment by ebndev** [comment]
Hi @Cheems-arch, welcome to the GitHub Community! 

I've gone ahead and moved this post to our New to GitHub category, as this topic might be more relevant there. Feel free to explore that category as there are many new users discovering how to use our platform as well.

**Reply by ebndev** [reply]
Also, our DevRel team put together a great series titled: [🎥 Let's learn git and GitHub together! ](https://github.com/orgs/community/discussions/126301) that can help you get started on the basics.

Best of luck on your journey 🌟

**Reply by Cheems-arch** [reply]
Wow, Thank you so much, I did not expect this reply.

**Comment by mustafacco7** [comment]
> ### حدد مجال الموضوع
> سؤال
> 
> ### جسم
> لقد بدأت جيثب. أنا لا أعرف حقا أي شيء. هل هناك بعض الأدوات عبر الإنترنت. هل تساعد في المبتدئين.

**Comment by Smudgeck1** [comment]
I don't understand any of it
